setup.py

```diff
@@ -98,15 +98,20 @@
                     os.path.relpath(root, PACKAGENAME), filename))
 package_info['package_data'][PACKAGENAME].extend(c_files)
 
+setup_requires = metadata.get('requires-dist')
+if setup_requires is None:
+    setup_requires =  ['numpy', 'cython', 'astropy']
+else:
+    # Convert the config setup_requires string into a list
+    setup_requires = setup_requires.strip().split()
 # Note that requires and provides should not be included in the call to
 # ``setup``, since these are now deprecated. See this link for more details:
 # https://groups.google.com/forum/#!topic/astropy-dev/urYO8ckB2uM
-
 setup(name=PACKAGENAME,
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=['astropy'],
+      setup_requires=setup_requires,
```

Comment:
I don't really understand what's going on here anymore.  `setup_requires` and `install_requires` are totally different things, nor should we be imposing the above defaults.